dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventChangeLogService.java

```diff
@@ -51,17 +50,6 @@ Page<EventChangeLog> getEventChangeLog(
       UID event, EventChangeLogOperationParams operationParams, PageParams pageParams)
       throws NotFoundException, ForbiddenException;
 
-  /**
-   * @deprecated use {@link EventChangeLogService#getEventChangeLog} instead
-   */
-  @Deprecated(since = "2.41")
```

Comment:
Are you going to remove the `TrackedEntity` from the remaining method names in here?

Reply:
The methods `deleteTrackedEntityDataValueChangeLog` and `deleteTrackedEntityDataValueChangeLog` I want to keep because there could potentially be an entry in the old table holding a foreign key to an event, so we would need to delete it as we've been doing so far.

I thought it made sense to keep the same name, so the distinction between the new and old logic is clear.